### PR TITLE
[FIX] website: batch of fixes

### PR DESCRIPTION
**Pull request by detrouxdev**
Batch of fixes related to the move of website edition in the backend

### CODE REVIEW

**Comment by detrouxdev** [comment]
@qsm-odoo I think this batch of fix is ready 👍🏻

File: addons/website/static/src/js/editor/snippets.options.js [hunk 1]
```diff
@@ -984,12 +984,13 @@ options.registry.OptionsTab = options.Class.extend({
         const hues = [];
         const saturationDiffs = [];
         let oneHasNoSaturation = false;
+        const baseStyle = getComputedStyle(document.documentElement);
```
Comment (qsm-odoo) [inline addons/website/static/src/js/editor/snippets.options.js:987]: Commit messages: don't end your title with a point + there are typos + you mentoin a commit which does not really do what you explain

File: addons/website/static/src/js/editor/snippets.options.js [hunk 2]
```diff
@@ -1493,7 +1493,8 @@ options.registry.Carousel = options.Class.extend({
      * @override
      */
     start: function () {
-        this.$target.carousel('pause');
+        this.$iframeTarget = this.ownerDocument.defaultView.$(this.$target[0]);
+        this.$iframeTarget.carousel('pause');
```
Comment (qsm-odoo) [inline addons/website/static/src/js/editor/snippets.options.js:1497]: I don't understand those lines, is that a common thing you had to do ? :thinking: I don't understand how `$iframeTarget` is different from `$target `

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: addons/website/static/src/js/editor/snippets.options.js (hunk 2 above)
Comment (detrouxdev) [reply addons/website/static/src/js/editor/snippets.options.js]: I changed $iframeTarget to $bsTarget which I made available to all snippets.options via include. 😬 There might be a better way to do that, or maybe it's even best to do it locally, I'm really not sure.

I also tried data-bs-no-jquery on the frontend and it did break some features of the frontend, and prevented the edition to start, so I thought that for the scope of fixing this, I wouldn't do the changes here.

Here's what I found though: Disabling bootstrap's jQuery mainly impact website_sale, website_slides and the edition code to handle mega menus and dropdowns. Some of the challenges might be that it's not longer possible to access batches of widgets such as doing `$('.popover').popover('hide')` which I have seen at least once.

Probably not necessary but if needed I can push a branch on odoo-dev with my experiments (I tried adapting the code for the website_sale cart dropdown).

**Review by qsm-odoo — APPROVED** [review]
@detrouxdev I made this diff: https://github.com/odoo/odoo/compare/f08a8f5b5ed04d10e08e29af962fcda488baa988..d9aa644b22a93e6da27309f922751e482740369f
You probably did not know about:
- `console.error` in tour tests
- `this.$anchor` in `run`

Thanks for the fixes :+1: 

@robodoo rebase-ff r+

**Pull request closed without merging**